media/css/firefox/new/fx-lifestyle/base.less

```diff
@@ -0,0 +1,292 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import '../../../sandstone/lib.less';
+
+@blue: #0a99d5;
+@grey: #EFEFEF;
+
+#outer-wrapper {
+    background: @grey;
+}
+
+#wrapper {
+    background: #fff;
+    width: auto;
+
+    // add icon to download buttons *not* in family nav
```

Comment:
Remnants of initial commit - used to have styles for family nav, which was never displayed. 🤷‍♂️